### PR TITLE
Bump storybook to 5.2.6 to fix local performance issues

### PR DESCRIPTION
**Pull request by dominickendrick**
## What does this change?
I was getting around 50% cpu usage when storybook was idle on my local machine. I found this issue https://github.com/storybookjs/storybook/issues/8529 so I've bumped the version of storybook and this fixes the problem
### Before
Screen Shot 2020-06-04 at 11 41 34

### After
Screen Shot 2020-06-04 at 12 13 51

## Why?
Better developer experience ;)

### CODE REVIEW

**Comment by oliverlloyd** [comment]
Hmm, bit odd that that Chromatic thinks all the snapshots are new though, that feels off. Is it maybe related to the package conflicts?

**Comment by liywjl** [comment]
I had a feeling things were getting slow when running storybook, thank you for the fix!

**Comment by dominickendrick** [comment]
@liywjl Any idea why the `cypress` step is failing on team city ?

**Comment by liywjl** [comment]
@dominickendrick looks like its an issue with one of the APIs (`/most-read-geo`) running slower than usual

File: package.json [hunk 1]
```diff
@@ -105,12 +105,13 @@
...
-        "@storybook/addon-viewport": "^5.2.5",
-        "@storybook/addons": "^5.2.5",
-        "@storybook/react": "^5.2.5",
+        "@babel/preset-typescript": "^7.3.3",
+        "@storybook/addon-actions": "^5.2.6",
+        "@storybook/addon-links": "^5.2.6",
+        "@storybook/addon-storysource": "^5.2.6",
+        "@storybook/addon-viewport": "^5.2.6",
+        "@storybook/addons": "^5.2.6",
+        "@storybook/react": "^5.2.6",
```
Comment (ndelangen) [inline package.json:114]: This actually upgrades to 5.3.19, see the lockfile.

**Comment by gtrufitt** [comment]
Closing this in favour of https://github.com/guardian/dotcom-rendering/pull/1705 for now

**Pull request closed without merging**